src/puppetlabs/puppetdb/scf/migrate.clj

```diff
@@ -727,7 +727,7 @@
              :values facts
              :timestamp timestamp
              :environment environment
-             :producer_timestamp nil}
+             :producer_timestamp timestamp}
```

Comment:
I don't think this will work because we need to head off people who already got through this migration with nils